VideoWeb/VideoWeb/ClientApp/src/app/shared/error/error.component.html

```diff
@@ -45,8 +45,9 @@ <h1 class="govuk-heading-xl">{{'error.problem-with-connection' | translate}}</h1
                     type="button"
                     (click)="reconnect()"
                     [attr.aria-label]="'error.button-reconnect' | translate">
-                {{'error.button-reconnect' | translate}} 
+                {{'error.button-reconnect' | translate}}
             </button>
+            <p *ngIf="failedAttemptToReconnect" id="failed-connect-ui">Internet connection detected: <strong [ngStyle]="{'color':connectionStatus?'green':'red'}">{{connectionStatus}}</strong></p>
```

Comment:
Just wondering if we need to replace that raw "Internet connection detected : " text with a translation so it will work in Welsh:

`<p *ngIf="failedAttemptToReconnect" id="failed-connect-ui">{{error.connection-status}}<strong [ngStyle]="{'color':connectionStatus?'green':'red'}">{{connectionStatus}}</strong></p>`

(or something similar)